api/routes/user.js

```diff
@@ -0,0 +1,118 @@
+// Layer 1: Routes
+import { Router } from 'express';
+import { v4 as uuidv4 } from 'uuid'
+import Joi from 'joi'
+
+import userServices from '../../services/user.service.js'
+const userRoute = Router();
+
+const schema = Joi.object({
+    login: Joi.string()
+        .alphanum()
+        .min(3)
+        .max(30)
+        .required(),
+
+    password: Joi.string()
+        .pattern(/^(?=.*[A-Za-z])(?=.*\d)[A-Za-z\d]{8,}$/),
+
+    age: Joi.number()
+        .integer()
+        .min(4)
+        .max(130),
+
+    isDeleted: Joi.boolean().required()
+})
+
+
+userRoute.get('/', (req, res) => {
+    userServices.getAllUsers()
+        .then(users => {
+            res.json(users);
+        })
+        .catch(err => {
+            res.status(500).send(err);
+        });
+});
+
+userRoute.get('/users', (req, res) => {
+    userServices.getAllUsers()
+        .then(users => {
```

Comment:
same for all instances below.

Reply:
Thanks, I will check all the instance and update.